tests/testthat/test-ui-heatmap.R

```diff
@@ -42,6 +42,8 @@ test_that("cXheatmap9", {
 
 test_that("cXheatmap10", {
     check_ui_test(cXheatmap10())
+
+    message('The plot points fade on clicking the legends - working as designed')
```

Comment:
I don't think that this is specific enough to explain what we are making sure the tester understands is working as designed.  How about 'Plot points all fade on selecting shapeBy and sizeBy legend items - working as designed.'

Reply:
done